Common/Transforms/itkAdvancedCombinationTransform.hxx

```diff
@@ -1034,16 +1034,15 @@ AdvancedCombinationTransform<TScalarType, NDimensions>::GetJacobianOfSpatialJaco
   JacobianOfSpatialJacobianType & jsj,
   NonZeroJacobianIndicesType &    nonZeroJacobianIndices) const
 {
-  SpatialJacobianType           sj0;
-  JacobianOfSpatialJacobianType jsj1;
+  SpatialJacobianType sj0;
   m_InitialTransform->GetSpatialJacobian(inputPoint, sj0);
   m_CurrentTransform->GetJacobianOfSpatialJacobian(
-    m_InitialTransform->TransformPoint(inputPoint), jsj1, nonZeroJacobianIndices);
+    m_InitialTransform->TransformPoint(inputPoint), jsj, nonZeroJacobianIndices);
 
   jsj.resize(nonZeroJacobianIndices.size());
```

Comment:
This `jsj.resize(nonZeroJacobianIndices.size())` may no longer be necessary, but it's hard for me decide, just by looking at those few lines of code.